.travis.yml

```diff
@@ -8,10 +8,20 @@ go:
 install:
         - mkdir -p $HOME/gopath/src/unidoc/
         - ln -s $HOME/gopath/src/github.com/unidoc/unioffice $HOME/gopath/src/github.com/unidoc/unioffice
+        - mkdir -p $HOME/dotnet
+        - cd $HOME/gopath/src/github.com/unidoc/unioffice
+        - mv aspnetcore.tar.gz $HOME/dotnet
```

Comment:
Can you put aspnetcore.tar.gz and the validator into a dedicated folder, such as ".travis"  or ".ci" or something like that

Reply:
Best would be to download these files as part of the CI process rather than keeping it in the unioffice repository

Reply:
ok, will do

Reply:
fixed